doc/gallery/introduction/pytensor_intro.ipynb

```diff
@@ -0,0 +1,3311 @@
+{
```

Comment:
<p>Remove "It's completely ours! (And nobody else uses it)", since this is for broader consumption :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pytensor/pull/1585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='107'/>

Reply:
Removed it